.ci/buildci.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# this script check for 'Circle' in the commit message
+
+MESSAGE=$(git log -1 HEAD --pretty=format:%s)
+
+if [[ "$MESSAGE" == *Circle* ]]; then
```

Comment:
So are you checking if there is `Circle` anywhere in the commit message?
https://github.com/coala/coala-bears/issues/1946#issuecomment-317456449
https://github.com/coala/coala-bears/issues/1946#issuecomment-317467859
I don't think the above comments suggest that. Maybe they didn't reach a decision over there. You should discuss it with them.

Reply:
It is just a way to limit circle CI builds on the basis of condition which could be any message or word (Circle for now) in the commit. I did ask for guidance before starting on this issue. 
For now i think i should wait for them to review it first and then I would discuss further. Your thoughts about this?
And thanks a lot for the suggestion on commit message :)